Allow to overwrite the default generated output/result filename

piotrbulinski opened the pull request.

Commit:
ignore PyCharm working directory

Commit:
allow to overwrite the result/output filename

ikeeip commented:
It is good idea! But there is a problem with this patch. The "--result-filename" option will not work without "--destination-path" option. I think the good idea is make only one option "--output" ("-o") with default behavior to write output to stdout.

Thanks for contributing!

piotrbulinski commented:
I was thinking about just using --output/-o, but didn't want to mess too much with the code and its backward compatibility. If you are fine with replacing those two with a single output parameter, I can take care of that.

Commit:
replace destination-path and result-filename with single output parameter; add password example support

Commit:
describe -o/--ouput in the README file

piotrbulinski commented:
That's done. I've also added an example for password format (we are using it in our swagger file and swg2rst crashed since it was missing), also changed the -format from required, to having a default='rst' - since that's the only option for now, we can have a shorter command line.

I hope this pull request is not too messy (too many changes at the same time). Please let me know if that's ok or not (I can provide separate pull requests for each change).
